grudge/op.py

```diff
@@ -145,6 +161,56 @@ def local_grad(dcoll, vec):
     return _bound_grad(dcoll)(u=vec)
 
 
+def reference_derivative_matrices(actx, element_group):
```

Comment:
This file is getting big. Are you OK with adding sectioning comments?

Reply:
I split out the new operators into a separate `operators.py` file, and I've added sectioning comments there too